packages/js/src/classic-editor/components/metabox/index.js

```diff
@@ -0,0 +1,140 @@
+
+/**
+ * Creates the focus keyphrase input component.
+ *
+ * @param {string} focusKeyphraseInfoLink The URL for the help link.
+ *
+ * @returns {JSX.Element} The focus keyphrase input.
+ */
+const FocusKeyphraseInput = ( { focusKeyphraseInfoLink } ) => {
+	const focusKeyphrase = useSelect( select => select( SEO_STORE_NAME ).selectKeyphrase() );
+	const displayNoKeyphraseMessage = useSelect( select => select( EDITOR_STORE_NAME ).getSEMrushNoKeyphraseMessage() );
+	const displayNoKeyphraseForTrackingMessage = useSelect( select => select( EDITOR_STORE_NAME ).hasWincherNoKeyphrase() );
+	const isSEMrushIntegrationActive = useSelect( select => select( EDITOR_STORE_NAME ).getIsSEMrushIntegrationActive() );
+	const { updateKeyphrase } = useDispatch( SEO_STORE_NAME );
+	const { setMarkerPauseStatus } = useDispatch( EDITOR_STORE_NAME );
+
+	const handleFocusKeyphraseChange = useCallback( keyphrase => updateKeyphrase( { keyphrase } ), [ updateKeyphrase ] );
+
+	const pauseMarker = useCallback( () => setMarkerPauseStatus( true ), [ setMarkerPauseStatus ] );
+	const startMarker = useCallback( () => setMarkerPauseStatus( false ), [ setMarkerPauseStatus ] );
+
+	return (
+		<KeywordInput
+			keyword={ focusKeyphrase }
+			displayNoKeyphraseMessage={ displayNoKeyphraseMessage }
+			displayNoKeyphraseForTrackingMessage={ displayNoKeyphraseForTrackingMessage }
+			isSEMrushIntegrationActive={ isSEMrushIntegrationActive }
+			helpLink={ focusKeyphraseInfoLink }
+			onFocusKeywordChange={ useDebounce( handleFocusKeyphraseChange ) }
+			onFocusKeyword={ pauseMarker }
+			onBlurKeyword={ startMarker }
+		/>
+	);
+};
+
+FocusKeyphraseInput.propTypes = {
+	focusKeyphraseInfoLink: PropTypes.string.isRequired,
+};
+
+/**
+ * Creates the Metabox component.
+ *
+ * @returns {JSX.Element} The Metabox.
+ */
+const Metabox = () => {
+	const settings = useSelect( select => select( EDITOR_STORE_NAME ).getPreferences() );
+	const isKeywordAnalysisActive = useSelect( select => select( EDITOR_STORE_NAME ).getIsKeywordAnalysisActive() );
+	const baseUrl = useSelect( select => select( EDITOR_STORE_NAME ).getBaseUrlFromSettings() );
+	const shoppingData = useSelect( select => select( EDITOR_STORE_NAME ).getShoppingData() );
+	const siteIconUrl = useSelect( select => select( EDITOR_STORE_NAME ).getSiteIconUrlFromSettings() );
+	const previewImageUrl = useSelect( select => select( EDITOR_STORE_NAME ).getSnippetEditorPreviewImageUrl() );
+	const analysisType = useSelect( select => select( SEO_STORE_NAME ).selectAnalysisType() );
+
+	return (
+		<Fragment>
+			<SidebarItem
+				key="warning"
+				renderPriority={ 1 }
+			>
+				<Warning />
+			</SidebarItem>
+			{ isKeywordAnalysisActive && <SidebarItem key="keyword-input" renderPriority={ 8 }>
+				<FocusKeyphraseInput focusKeyphraseInfoLink={ wpseoAdminL10n[ "shortlinks.focus_keyword_info" ] } />
+				{ ! wpseoScriptData.metabox.isPremium && <Fill name="YoastRelatedKeyphrases">
```

Comment:
In an ideal situation these values are not retrieved from the window, but fine for now I think